### DISCUSSION TITLE
GSOC 2024 Contributor

### DISCUSSION THREAD

**Discussion by sssa209** [unicode-org/conformance#203] [Q&A]
Hi there,

I'm a computer science major college student currently studying in China of CS major. interested in this `Expand Unicode Conformance Test Suite with More Data and Implementations` project. I'd like to fully commit to it and apply my abilities and more contributions.
It'll be great if anyone could give me any advice on starting up. Will it be helpful to fix some starter issues?

Thank you for your time! Looking forward to building this fantastic project with you all!

**Comment by echeran** [comment]
Hi @Yoofin and thanks for showing interesting in contributing to this project! For this project, issue #213 seems like a good first issue to assess your ability to succeed as a contributor. Please take a look. If you have any questions before you begin, you can leave them in the issue. If you create a Pull Request, then the Pull Request is a more natural place to continue discussions and questions on specific details.

cc @sven-oly

**Reply by sssa209** [reply]
Sure! I'm currently working on it. Thank you for this guidance!
